bridge-ffi/include/sdk-core-bridge.h

```diff
@@ -0,0 +1,216 @@
+#pragma once
```

Comment:
Just double checking we want to check this in?

Reply:
This is a tough question. Usually in Rust, unlike Go, we don't want generated code in. However, when people are developing, they often want a header file already there. I think there's enough value to leave here.

We could have a CI check make sure it doesn't change on build from what is already in repo, but not sure this PR should wait on that.

Reply:
> We could have a CI check make sure it doesn't change on build from what is already in repo, but not sure this PR should wait on that.

Well, I may have found an easy way to do this, I will confirm and get back

Reply:
I have added this check. The BuildKite lint step will fail if the cbindgen generates something different than what's in repo with a:

> bridge-ffi's header file changed unexpectedly from what's on disk

Example at https://buildkite.com/temporal/core-sdk/builds/1150#a06e1d7b-d281-47bf-b24b-fbafa143945e

Reply:
:+1: